tracing.go

```diff
@@ -2,15 +2,22 @@
 package opencensus
 
 import (
+	"crypto/rand"
 	"encoding/base64"
+	"math/big"
+	"strconv"
 
 	"github.com/ThreeDotsLabs/watermill"
 	"github.com/ThreeDotsLabs/watermill/message"
 	"go.opencensus.io/trace"
 	"go.opencensus.io/trace/propagation"
 )
 
-const spanContextKey = "opencensus_span_context"
+const (
+	spanContextKey               = "opencensus_span_context"
+	spanEventIDKey               = "opencensus_event_id"
+	spanEventPayloadAttributeKey = "event-payload"
```

Comment:
Is there a reason for using mixed convention for key names (xxx-xxx vs xxx_xxx)?

Reply:
I will double-check that. In fact, the two top ones are context keys, while the third one is the span attribute key. The conventions may be different. Will get back with the results.

Reply:
https://github.com/czeslavo/watermill-opencensus/pull/6/commits/0c281d9a162452456cbca755e765cb33add72009